src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -349,7 +347,8 @@ class DemoPage extends StatelessWidget {
 }
 ```
 
-1. Run the app (or stop and restart it, if you already had it running
-   before adding the plugin). When you click the 'Show Flutter homepage'
-   you should see the phone's default browser open, and the Flutter
-   homepage appear.
\ No newline at end of file
+1. Run the app (or stop and restart it, if it was already running
```

Comment:
Like before, this reverts to #1. Indenting the previous code snippet should fix the problem.